hardware/PiGpio/36-rpi-gpio.html

```diff
@@ -70,8 +70,9 @@
 <script type="text/html" data-template-name="rpi-gpio in">
 
     <div class="form-row" style="min-width: 540px">
-        <label><i class="fa fa-circle"></i> <span data-i18n="rpi-gpio.pinname"></span></label>
-        <input type="text" id="node-input-pin" style="display:none;">
+        <label>&nbsp;&nbsp;&nbsp;&nbsp;<span data-i18n="rpi-gpio.pinname"></span></label>
```

Comment:
I think I'd prefer the new field to be arranged below the existing selector so that by the time someone scrolls down it may already match the pin they have selected. (and leave the fa-icon)

Reply:
I incorporated your suggestions in my second commit. I also closed validation escapes when directly typing the pin number.